src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -150,19 +181,25 @@ const mapStateToProps = state => {
     prescriptions,
     canAccessRx,
     authenticatedWithSSOe: isAuthenticatedWithSSOe(state),
+    canAccessMessaging: profileState.services.includes(
+      backendServices.MESSAGING,
+    ),
+    unreadMessagesCount: selectUnreadMessagesCount(state),
   };
 };
 
 const mapDispatchToProps = {
+  fetchFolder: fetchFolderAction,
   loadPrescriptions: loadPrescriptionsAction,
   fetchConfirmedFutureAppointments: fetchConfirmedFutureAppointmentsAction,
 };
 
 HealthCare.propTypes = {
   authenticatedWithSSOe: PropTypes.bool.isRequired,
-  isCernerPatient: PropTypes.bool.isRequired,
+  isCernerPatient: PropTypes.bool,
```

Comment:
On first render this is undefined, so needed to make this not required.

Reply:
hmm... so we could have a flash of showing the non-Cerner content if this prop eventually resolves to `true`?

Reply:
Currently, that is correct. I have another ticket open to add in a loading spinner to the Health Care section until all endpoints resolve, so when that is implemented this is no longer an issue.